ui/oopsyraidsy/buff_map.ts

```diff
@@ -47,51 +50,136 @@ export const missedEffectBuffMap: readonly MissableEffect[] = [
     effectId: 'A26',
     collectSeconds: 2,
   },
+  {
+    // BRD
+    id: 'Mage\'s Ballad',
+    type: 'damage',
+    effectId: '8A9',
+    collectSeconds: 45,
+  },
+  {
+    // BRD
+    id: 'Army\'s Paeon',
+    type: 'damage',
+    effectId: '8AA',
+    collectSeconds: 45,
+  },
+  {
+    // BRD
+    id: 'The Wanderer\'s Minuet',
+    type: 'damage',
+    effectId: '8A8',
+    collectSeconds: 45,
+  },
+  {
+    // DNC channeled heal
+    id: 'Improvisation',
+    type: 'heal',
+    effectId: 'A87',
+    collectSeconds: 15,
+  },
 ] as const;
 
 export const missedAbilityBuffMap: readonly MissableAbility[] = [
   {
+    // tank LB1
+    id: 'Shield Wall',
+    type: 'mitigation',
+    abilityId: 'C5',
+  },
+  {
+    // tank LB2
+    id: 'Stronghold',
+    type: 'mitigation',
+    abilityId: 'C6',
+  },
+  {
+    // PLD LB3
+    id: 'Last Bastion',
+    type: 'mitigation',
+    abilityId: 'C7',
+  },
+  {
+    // WAR LB3
+    id: 'Land Waker',
+    type: 'mitigation',
+    abilityId: '1090',
+  },
+  {
+    // DRK LB3
+    id: 'Dark Force',
+    type: 'mitigation',
+    abilityId: '1091',
+  },
+  {
+    // GNB LB3
+    id: 'Gunmetal Soul',
+    type: 'mitigation',
+    abilityId: '42D1',
+  },
+  {
+    // GNB
     id: 'Heart Of Light',
     type: 'mitigation',
     abilityId: '3F20',
   },
   {
+    // DRK
     id: 'Dark Missionary',
     type: 'mitigation',
     abilityId: '4057',
   },
   {
+    // WAR party shield + regen
     id: 'Shake It Off',
-    type: 'mitigation',
+    type: 'heal',
```

Comment:
```suggestion
    type: 'mitigation',
```

Yes it's a heal, but people use it as mitigation.

Reply:
It's a shield, it boosts effective HP.  Mitigations scale with incoming damage, heals and shields are always a fixed amount.

Reply:
Sure, but shields are mitigation too?  These types are just for icons and not anything else.  It's more about "what do you consider this ability", and I think divine veil and shake it off are more mitigation/shields and less heals even though they do both.

Reply:
If you're considering shields and mitigation to be the same then the SCH and SGE abilities are very inconsistent right now.

Reply:
That's fair! I guess I'm thinking of heals as being "top up" and shields/mit as being "prevent damage".  I only ever made oopsy icons for heal and shield and didn't separate into some third category of percent mitigation vs shields.